sst/test/src/tg5_benchmarks.cc

```diff
@@ -0,0 +1,39 @@
+#include "sstsimengtest.hh"
```

Comment:
I'm unsure as to whether we should be running binaries within the test suite. The other tests should be validating functionality, and the accuracy of a model is up to the user to validate.

Running benchmarks through SSTxSimEng to see if they work should be something our CI pipeline does rather than shipping binaries with SimEng and testing them through our test suite. 

@jj16791 @rahahahat @dANW34V3R your thoughts on this would be useful

Reply:
See [comment](https://github.com/UoB-HPC/SimEng/pull/271#discussion_r1016472564). If it is just to test that they work, shouldn't that be in the CI pipeline?

Reply:
I'd fall on the side of not having them in the test suite. The small assembly snippets should test all supported interactions and validate that the built integration works as expected. The running of benchmarks would be better suited to in-house testing (CI pipeline), where we ensure no emergent bugs are present from recent developments or new integration problems from actions such as updating SST.

Reply:
The work done to support the simulation of these benchmarks will be used in a CI pipeline so no work will go to waste. @rahahahat make sure you store those developments somewhere for use later this year.

Reply:
Sure sounds good will do.

Reply:
Done.

Reply:
@rahahahat @FinnWilkinson @dANW34V3R briefly unresolving to ensure I've understood the final actions of this comment. Am I correct in saying that we shouldn't be committing any work associated with running full workloads (e.g. tests, binaries, runners, stat comparators, etc)? I may have misunderstood what was intended here.

Reply:
In my opinion, yes you are correct. Running binaries through SSTxSimEng should be part of the CI pipeline testing (when updated) and not in this PR. From what I can tell Rahat has now removed everything?

Reply:
Yes, removal of tests, binaries and runners as these are just for our internal verification in the CI pipeline. Not sure if there were any specific comparators

Reply:
Yes everything has been removed.

Reply:
What about `sst/test/sstconfigs/fastL1ForBinaries.py`

Reply:
I've removed it now.